### DISCUSSION TITLE
近期上报的重点关注变异株序列（XBB.1.16* XBB.1.19.1 XBB.1.5* XBB.1.33* XCC* XBB.1.22.1*）不包括上海

### DISCUSSION THREAD

**Discussion by Memorablea** [Memorablea/COVID-19-#20] [General]
-4月上传至GISAID汇总（4.17）
EG.1上海EPI_ISL_17485625与福建福州17485548
EL.1上海EPI_ISL_17485622

**Comment by Memorablea** [comment]
- 4月19日上传至GenBase的序列
格式为C_AA00XXXX.1

- XBB.1.16.1
9614-9615、9616-9618（均为浙江的印度尼西亚输入），9624（浙江）

- XBB.1.16
9695（浙江），9690（浙江的日本输入）

**Comment by Memorablea** [comment]
![image](https://user-images.githubusercontent.com/125779492/233398667-ad9f6a03-d9dc-422f-9169-cb3342fd9392.png)

- XBB.1.5.24
C_AA010800.1、C_AA010800.1-C_AA010782.1，C_AA010775.1（江苏）

**Comment by Memorablea** [comment]
> * 4月19日上传至GenBase的序列
>   格式为C_AA00XXXX.1
> * XBB.1.16.1
>   9614-9615、9616-9618（均为浙江的印度尼西亚输入），9624（浙江）
> * XBB.1.16
>   9695（浙江），9690（浙江的日本输入）

国家疾控局回应XBB.1.16流行情况
我国近期输入病例中XBB.1.16的占比增加。我国本土病例中XBB.1.16仍维持极低水平，未形成传播优势。

**Comment by Memorablea** [comment]
- 4月23日更新
- XBB.1.19.1
  来自：上海、新疆、乌鲁木齐、武汉和杭州（[C_AA009132.1、C_AA011023.1、EPI_ISL_17494041、EPI_ISL_17501741、EPI_ISL_17514261）

- XBB.1.5.24
  来自安徽苏州（EPI_ISL_17516743）

- XBB.1.16
  来自浙江（|EPI_ISL_17501475、EPI_ISL_17501572）

- XBB.1.16.1
  来自浙江（EPI_ISL_17501577、EPI_ISL_17501521-23、EPI_ISL_17501553、56、77）
![image](https://user-images.githubusercontent.com/125779492/233821874-838fd4bc-3c29-401a-82ae-2b24fd82f923.png)

**Comment by Memorablea** [comment]
- XCC——XBB.1.9.1与CH.1.1.1重组株
- 浙江EPI_ISL_17487850
![image](https://user-images.githubusercontent.com/125779492/233837663-61190721-18a4-4746-bca8-523b5586c6f6.png)

**Comment by Memorablea** [comment]
- XBB.1.5.46
  来自上海（上海/SCDC-2211/2023，XBB.1.5.46）
![image](https://user-images.githubusercontent.com/125779492/233905933-dab3b8b0-7fa2-46ba-9fee-a9c3c0c13114.png)

**Comment by Memorablea** [comment]
4月29日更新来自4月26、27日上报的序列
- XBB.2.3
C_AA011841.1（福建，新加坡输入）、C_AA011791.1，C_AA011772.1（云南）

- XBB.1.16
C_AA011839.1，C_AA011822.1（福建）、C_AA011770.1（云南）、C_AA011746.1，C_AA011745.1，C_AA011744.1（四川遂宁）、 C_AA011640.1，C_AA011639.1（湖北天门，德国输入）

- XBB.1.16.1
C_AA011607.1（湖北黄冈，台湾输入）

- XBB.1.11.1
C_AA011782.1、C_AA011781.1（云南）

- XBB.1.22.1
C_AA011642.1-C_AA011653.1（湖北武汉）、 C_AA011610.1，C_AA011613.1（湖北黄石）、C_AA011631.1-C_AA011633.1（湖北荆门）、C_AA011606.1，C_AA011605.1（湖北黄冈）

**Comment by Memorablea** [comment]
- XBB.1.16
 C_AA002464.1/EPI_ISL_17587053，C_AA011822.1/EPI_ISL_17547639（云南）、EPI_ISL_17584631，EPI_ISL_17584672，EPI_ISL_17584525，EPI_ISL_17584664—EPI_ISL_17584666，EPI_ISL_17584648，EPI_ISL_17584678，EPI_ISL_17584611，EPI_ISL_17584728（广东）、EPI_ISL_17563551（四川）
- XBB.1.19.1
EPI_ISL_17592189（北京）
- XBB.1.16.1
EPI_ISL_17592172，EPI_ISL_17592200（北京）
- XBB.1.17.1
EPI_ISL_17592202（北京）、EPI_ISL_17584631（广州）
- XBB.1.18.1
EPI_ISL_17584991，EPI_ISL_17584994，EPI_ISL_17584996，EPI_ISL_17584995，EPI_ISL_17584997，EPI_ISL_17584993，EPI_ISL_17584990（甘肃白银）

**Comment by Outpfmance** [comment]
近期XBB*在中国开始爆发，并且始终保持多分支快速增长，这个议题貌似已经没必要继续存在了

**Comment by Memorablea** [comment]
> * XBB.1.16
>   C_AA002464.1/EPI_ISL_17587053，C_AA011822.1/EPI_ISL_17547639（云南）、EPI_ISL_17584631，EPI_ISL_17584672，EPI_ISL_17584525，EPI_ISL_17584664—EPI_ISL_17584666，EPI_ISL_17584648，EPI_ISL_17584678，EPI_ISL_17584611，EPI_ISL_17584728（广东）、EPI_ISL_17563551（四川）
> * XBB.1.19.1
>   EPI_ISL_17592189（北京）
> * XBB.1.16.1
>   EPI_ISL_17592172，EPI_ISL_17592200（北京）
> * XBB.1.17.1
>   EPI_ISL_17592202（北京）、EPI_ISL_17584631（广州）
> * XBB.1.18.1
>   EPI_ISL_17584991，EPI_ISL_17584994，EPI_ISL_17584996，EPI_ISL_17584995，EPI_ISL_17584997，EPI_ISL_17584993，EPI_ISL_17584990（甘肃白银）

转移至讨论区
